src/electron/main.ts

```diff
@@ -508,7 +507,7 @@ async function createWindow(): Promise<void> {
 		minWidth: 780,
 		minHeight: 380,
 		titleBarStyle: 'hiddenInset',
-		backgroundColor: Color.Grey97,
+		backgroundColor: '#f7f7f7',
```

Comment:
Some special reason for this one?

Reply:
Color.Grey97 didn’t work (I guess because electron only allows hex values)

Reply:
Ah, so lets use the `color` module here to convert from rgb to hex then.